adhocracy4/polls/models.py

```diff
@@ -92,6 +165,10 @@ class Vote(UserGeneratedContentModel):
         related_name='votes'
     )
 
+    @property
+    def is_other_vote(self):
```

Comment:
elsewhere we say `has_other_vote` in cases like that. But this would also be fine to merge!